### PR TITLE
Removed all obsolete UNIFIED symbol checks

### PR DESCRIPTION
**Pull request by Qonstrukt**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Remove all obsolete UNIFIED symbol checks, as requested in #1222.

**What is the current behavior? (You can also link to an open issue here)**
There's checks for whether the unified framework is used.

**What is the new behavior (if this is a feature change)?**
These checks are no longer required, as it's no longer supported.

**Does this PR introduce a breaking change?**
Nope.

**Please check if the PR fulfills these requirements**
- [x] The commit follows our guidelines: https://github.com/reactiveui/reactiveui#contribute
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:
Can't really check if the Mac version builds, Xamarin Studio seriously chokes on the Mac library projects. Can't find references to AppKit etc. So I'm hoping the build works.

I've also removed a lot of unused using statements. And maybe some old code standards have been refactored by my IDE settings. But they should comply with the latest standards used throughout the project.

### CODE REVIEW

File: src/ReactiveUI/Cocoa/CommonReactiveSource.cs [hunk 1]
```diff
@@ -226,117 +213,110 @@ private void SubscribeToSectionInfoChanges(int sectionInfoId, IReadOnlyList<TSec
...
 
-                        this.Log().Debug("[#{0}] SectionInfo is changing.", sectionInfoId);
-                        sectionInfoDisposable.Disposable = Disposable.Empty;
-                    }));
+                    this.Log().Debug("[#{0}] SectionInfo is changing.", sectionInfoId);
+                    sectionInfoDisposable.Disposable = Disposable.Empty;
+                }));
 
             disposables.Add(
-                sectionChanged.Subscribe(x =>
```
Comment (Qonstrukt) [inline src/ReactiveUI/Cocoa/CommonReactiveSource.cs]: Correct, some indenting changed automatically.

File: src/ReactiveUI/Cocoa/ReactiveNSViewController.cs [hunk 2]
```diff
@@ -31,10 +20,7 @@ namespace ReactiveUI
     /// (i.e. you can call RaiseAndSetIfChanged)
     /// </summary>
     public class ReactiveViewController : NSViewController,
-        IReactiveNotifyPropertyChanged<ReactiveViewController>, IHandleObservableErrors, IReactiveObject
-#if UIKIT
```
Comment (Qonstrukt) [inline src/ReactiveUI/Cocoa/ReactiveNSViewController.cs]: Yeah I checked whether or not this should be able to work, and all the functionality seems to be there actually. So I didn't see a reason to leave the check for `UIKIT` there.

**Comment by Qonstrukt** [comment]
I'll check csproj and nuspec files as well, thanks for pointing that out. I did a quick search in the whole project, but I might've filtered some out.

**Comment by Qonstrukt** [comment]
@reactiveui/reviewers-mac is there someone that could test this in an existing Mac project. I will set one up myself, especially to check whether `ICanActivate` works in `NSViewController`. But there might be other gotcha's that influence existing projects.

**Comment by Qonstrukt** [comment]
I notice the formatting in the csproj files got messed up. I'll look at this tonight and finish up a test on Mac as well.

**Comment by olevett** [comment]
Sorry for my slow reply here, I think it's likely to be a painful merge either way, I'm happy to tackle it from the CSPROJ side of things so think this could be merged first - depends on what @ghuntley and @kentcb think!

**Comment by ghuntley** [comment]
My thoughts is this should be merged after the last v7.x release goes out. Wether it's done before @olevett's PR at #1332 is your call. My gut is divided whether to do it before or after. It definitely needs to be done as part of v8 / netstandard release.

**Pull request merged**